lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -171,6 +194,47 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
     """
   end
 
+  defp edit_sessions(assigns) do
+    ~H"""
+    <div class="mx-4 mr-0 text-gray-600 flex flex-row gap-1">
+      <%= if @editing? do %>
+        <.menu id="edit-sessions">
+          <:toggle>
+            <button class="button-base button-outlined-gray px-4 py-1">
+              <span>Actions</span>
+              <.remix_icon icon="arrow-down-s-line" class="text-lg leading-none align-middle ml-1" />
+            </button>
+          </:toggle>
+          <:content>
+            <%= for action <- ["cancel_bulk_edit", "select_all"] do %>
+            <a href="#" class="menu-item text-gray-600" phx-click={action}>
+              <.remix_icon icon={action_icon(action)} />
+              <span class="font-medium"><%= action_label(action) %></span>
+            </a>
+            <% end %>
+            <%= for action <- ["close_all", "disconnect"] do %>
+              <%= live_patch to: Routes.home_path(@socket, :edit_sessions, action),
+              class: "menu-item
+                #{if length(@selected_sessions) == 0, do: "opacity-50 pointer-events-none"}
+                #{if action == "close_all", do: "text-red-600", else: "text-gray-600"}",
```

Comment:
I think it may end up simpler to just render the items separately rather than using loops. It is a bit of repetition, but should give more control.

Sidenote: please @josevalim correct me if I'm wrong, but static loops like that will make LV send more data over the wire, since more of the rendered content is dynamic?

Reply:
Yes, loops means more is rendered when any of the assigns inside change. I agree explicitly rendering the actions would be clearer too. :)

Reply:
Thanks! Nice til ❤️